public/app/features/scenes/components/Scene.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { PageToolbar, ToolbarButton } from '@grafana/ui';
+
+import { SceneObjectBase } from '../core/SceneObjectBase';
+import { SceneComponentProps, SceneObjectState, SceneObject } from '../core/types';
+import { UrlSyncManager } from '../services/UrlSyncManager';
+
+interface SceneState extends SceneObjectState {
+  title: string;
+  layout: SceneObject<any>;
+  actions?: Array<SceneObject<any>>;
+  isEditing?: boolean;
+}
+
+export class Scene extends SceneObjectBase<SceneState> {
+  static Component = SceneRenderer;
+  urlSyncManager?: UrlSyncManager;
+
+  onMount() {
+    super.onMount();
+    this.urlSyncManager = new UrlSyncManager(this);
+  }
+
+  onUnmount() {
+    super.onUnmount();
+    this.urlSyncManager!.cleanUp();
+  }
+}
+
+function SceneRenderer({ model }: SceneComponentProps<Scene>) {
+  const { title, layout, actions = [], isEditing, $editor } = model.useMount().useState();
+
+  console.log('render scene');
+
+  return (
+    <div style={{ height: '100%', display: 'flex', flexDirection: 'column', flex: '1 1 0', minHeight: 0 }}>
+      <PageToolbar title={title}>
+        {actions.map((action) => (
+          <action.Component key={action.state.key} model={action} />
+        ))}
```

Comment:
think this is maybe better defines as "toolbarActions", and since the toolbar is part of the Grafana chrome (esp in the new nav design the toolbar actually lives outside the main route).  So this cannot really be part of the scene  in a layout sense. 

I think for nested scenes they will have to be part of the layout in a way. I think a nested / inner scene will be a different type then the top level